### PR TITLE
Issue 576

### PR DESCRIPTION
**Pull request by imd1**
Work complete on issue 576. Unit tests added and pass.

### CODE REVIEW

File: vsg/tests/generic/test_rule_600.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        self.assertEqual(oRule.identifier, '600')
+
+        lExpected = [14, 15]
+
+        oRule.analyze(self.oFile)
+        self.assertEqual(lExpected, utils.extract_violation_lines_from_violation_object(oRule.violations))
+
+    def test_rule_600_w_uppercase(self):
+        oRule = generic.rule_600()
+        oRule.suffixes = ['_G']
```
Comment (jeremiah-c-leary) [inline vsg/tests/generic/test_rule_600.py:33]: Could you make this '_W' and update the lExpected?

**Comment by jeremiah-c-leary** [comment]
Hey @imd1 ,

It looks really good.  I just had the one comment.

--Jeremy

File: vsg/rules/process/rule_600.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+class rule_600(token_suffix):
+    '''
+    Constant rule 600 checks for suffixes in process labels.
+    '''
+
+    def __init__(self):
+        token_suffix.__init__(self, 'process', '600', lTokens)
+        self.suffixes = ['_proc']
+        self.solution = 'Process labels'
```
Comment (imd1) [inline vsg/rules/process/rule_600.py:19]: I was going to ask you exactly what these lines do in each rule, as I'm not confident they are consistent across the new rules I added, and some of the existing rule .e.g some do not have `solution`

Comment (imd1) [reply vsg/rules/process/rule_600.py:19]: Maybe you can review these lines in each rule to double check?

Comment (jeremiah-c-leary) [reply vsg/rules/process/rule_600.py:19]: I reviewed the `token_suffix.py` file and `self.solution` is not used.  I was at one point, but line 51 generates the solution based on the token being suffixed and the possible suffixes defined the by rule.  I must not have cleaned up the other rules.

Comment (jeremiah-c-leary) [reply vsg/rules/process/rule_600.py:19]: The suffix rule uses `self.suffixes` in its checks.

File: vsg/tests/generic/test_rule_600.py (hunk 1 above)
Comment (imd1) [reply vsg/tests/generic/test_rule_600.py:33]: i have renamed the test...

**Comment by jeremiah-c-leary** [comment]
Are we ready to merge this?

**Comment by imd1** [comment]
> Are we ready to merge this?

Yes

**Pull request merged**